code/upaya/app/controllers/devise/account_type_controller.rb

```diff
@@ -43,7 +43,7 @@ def unconfirmed_representative?
     def success
       redirect_to dashboard_index_path,
                   notice: t('upaya.notices.account_created',
-                            date: (Time.now + 1.year).strftime('%B %d, %Y'))
+                            date: (Time.current + 1.year).strftime('%B %d, %Y'))
```

Comment:
at that level of granularity, agreed that timezones are not an issue here.

what we do with C2 is use the https://github.com/kbaum/browser-timezone-rails gem to always present timestamps in the tz of the browser, even if they are always stored as UTC. probably not an issue here.